sdk/cosmos/azure-cosmos-kafka-connect/doc/configuration-reference.md

```diff
@@ -2,64 +2,64 @@
+| `azure.cosmos.auth.type`                                          | `MasterKey`  | There are two auth types are supported currently: `MasterKey`(PrimaryReadWriteKeys, SecondReadWriteKeys, PrimaryReadOnlyKeys, SecondReadWriteKeys), `ServicePrincipal`                                                                                                                                                                                                                                                                                                                         |
+| `azure.cosmos.account.key`                                        | `""`         | Cosmos DB Account Key (only required in case of `auth.type` as `MasterKey`)                                                                                                                                                                                                                                                                                                                                                                                                                    |
+| `azure.cosmos.auth.aad.clientId`                                  | `""`         | The clientId/ApplicationId of the service principal. Required for `ServicePrincipal` authentication.                                                                                                                                                                                                                                                                                                                                                                                           |
+| `azure.cosmos.auth.aad.clientSecret`                              | `""`         | The client secret/password of the service principal.                                                                                                                                                                                                                                                                                                                                                                                                                                           |
+| `azure.cosmos.mode.gateway`                                       | `false`      | Flag to indicate whether to use gateway mode. By default it is false.                                                                                                                                                                                                                                                                                                                                                                                                                          |
```

Comment:
we should add the documentation that by default it is DIRECT mode. But since direct mode is not a concept for kafka users, we should also point to this document - https://learn.microsoft.com/azure/cosmos-db/nosql/sdk-connection-modes

Reply:
Added